corehq/apps/app_manager/templates/app_manager/ng_partials/base_summary_view.html

```diff
@@ -35,4 +35,4 @@
       </div>
   </div>
 </script>
-{% endangularjs %}
```

Comment:
yea i actually think this is an issue with the diff? it exists on my branch locally